src/Form/SettingsForm.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\devel\Form\SettingsForm.
+ */
+
+namespace Drupal\logs_http\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Symfony\Component\HttpFoundation\Request;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Logger\RfcLogLevel;
+
+/**
+ * Defines a form that configures devel settings.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'logs_http_admin_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'logs_http.settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, Request $request = NULL) {
+    $form['logs_http_enabled'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Logs HTTP API'),
+      '#description' => t('Enable Logs HTTP POST'),
+      '#default_value' => \Drupal::state()->get('logs_http_enabled', TRUE),
```

Comment:
Look [here](https://github.com/Gizra/message/blob/8.x-1.x/src/Form/MessageSettingsForm.php#L94) how we used the config schema for that.